types/usb/index.d.ts

```diff
@@ -8,6 +8,10 @@
 
 import { EventEmitter } from "events";
 
+export class libusbException extends Error {
```

Comment:
Please follow the capitalisation naming convention for classes used elsewhere in this file:

`LibUSBException`